db/view/view_update_generator.cc

```diff
@@ -184,6 +189,12 @@ future<> view_update_generator::start() {
                     vug_logger.warn("Moving {} from staging failed: {}:{}. Ignoring...", s->ks_name(), s->cf_name(), std::current_exception());
                 }
                 _registration_sem.signal(num_sstables);
+
+                auto end_time = db_clock::now();
+                auto duration = std::chrono::duration<float>(end_time - start_time);
+                vug_logger.info("Processed {}.{}: {} sstables in {}ms = {}", s->ks_name(), s->cf_name(), sstables.size(),
+                                std::chrono::duration_cast<std::chrono::milliseconds>(duration).count(),
+                                utils::pretty_printed_throughput(input_size, duration));
```

Comment:
We should integrate this with the task manager, and then operators can view the progress any time, not just at the end. This code is fine as an intermediary until that happens, especially if we want to backport it.is

Reply:
Tip: to count milliseconds write `duration / 1ms` instead of that huge monster.

Reply:
> Tip: to count milliseconds write `duration / 1ms` instead of that huge monster.

Hahah, thanks.